Start-Up.md

```diff
@@ -0,0 +1,27 @@
+# Start-Up Procedure
+
+A quick-start procedure for starting up the AV-42C.
+
+<!-- TODO: put a picture here with a general layout and a note that it is subject to change -->
+
+On the right panel:
+
+1. Flip "Main Battery" switch to "On"
+2. Flip "Brake Lock" switch to "On" (this is so you don't accidentally roll while starting up)
+3. Flip "APU" switch to "On" (it is a covered switch, so you need to lift the cover first)
+4. Wait for the RPM gauge under the "APU" switch to spool up
+5. Flip the covered switch for one of the engines to "On" (you can try both at once, but you are likely to have start failures)
+6. Wait for the engine to spool up (RPM gauges for the engines are on the front panel on the right side)
+7. Flip the covered switch for the other engine to "On" and wait for it to spool up
+8. Flip "APU" switch to "Off" (you don't need it once the engines are running)
+
+Front panel:
```

Comment:
Perhaps we should have two sections here, instead: the one above for *Starting the Engines* and a section here for *Getting the Jet Mission-Ready* with a note that the steps aren't required and may be mission-specific or personal preference, that can include turning on MFDs, turning on the HMCS, etc.

Reply:
I agree. Have you looked at the comments in the file itself about referencing other files? It should probably be rolled into those. Maybe a "General Mission Readiness" and then specifics for different kinds of take-offs.

We don't want to split things into *too* many files in my opinion though, lots of unnecessary clicking between pages.

Reply:
> We don't want to split things into too many files in my opinion though, lots of unnecessary clicking between pages.
 Write

Oh agreed. I didn't mean separate pages, but different headers like:

```markdown
## Starting the Engines

1. Steps

## Getting the Jet Mission-Ready

2. Other steps
```

etc.